dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/AbstractServiceConfig.java

```diff
@@ -1,299 +1,300 @@
-        checkName(Constants.TOKEN_KEY, token);
-        this.token = token;
-    }
-
-    public Boolean isDeprecated() {
-        return deprecated;
-    }
-
-    public void setDeprecated(Boolean deprecated) {
-        this.deprecated = deprecated;
-    }
-
-    public Boolean isDynamic() {
-        return dynamic;
-    }
-
-    public void setDynamic(Boolean dynamic) {
-        this.dynamic = dynamic;
-    }
-
-    public List<ProtocolConfig> getProtocols() {
-        return protocols;
-    }
-
-    @SuppressWarnings({"unchecked"})
-    public void setProtocols(List<? extends ProtocolConfig> protocols) {
-        ConfigManager.getInstance().addProtocols((List<ProtocolConfig>) protocols);
-        this.protocols = (List<ProtocolConfig>) protocols;
-    }
-
-    public ProtocolConfig getProtocol() {
-        return CollectionUtils.isEmpty(protocols) ? null : protocols.get(0);
-    }
-
-    public void setProtocol(ProtocolConfig protocol) {
-        setProtocols(Arrays.asList(protocol));
-    }
-
-    @Parameter(excluded = true)
-    public String getProtocolIds() {
-        return protocolIds;
-    }
-
-    public void setProtocolIds(String protocolIds) {
-        this.protocolIds = protocolIds;
-    }
-
-    public String getAccesslog() {
-        return accesslog;
-    }
-
-    public void setAccesslog(Boolean accesslog) {
-        if (accesslog == null) {
-            setAccesslog((String) null);
-        } else {
-            setAccesslog(String.valueOf(accesslog));
-        }
-    }
-
-    public void setAccesslog(String accesslog) {
-        this.accesslog = accesslog;
-    }
-
-    public Integer getExecutes() {
-        return executes;
-    }
-
-    public void setExecutes(Integer executes) {
-        this.executes = executes;
-    }
-
-    @Override
-    @Parameter(key = Constants.SERVICE_FILTER_KEY, append = true)
-    public String getFilter() {
-        return super.getFilter();
-    }
-
-    @Override
-    @Parameter(key = Constants.EXPORTER_LISTENER_KEY, append = true)
-    public String getListener() {
-        return listener;
-    }
-
-    @Override
-    public void setListener(String listener) {
-        checkMultiExtension(ExporterListener.class, "listener", listener);
-        this.listener = listener;
-    }
-
-    public Boolean isRegister() {
-        return register;
-    }
-
-    public void setRegister(Boolean register) {
-        this.register = register;
-    }
-
-    public Integer getWarmup() {
-        return warmup;
-    }
-
-    public void setWarmup(Integer warmup) {
-        this.warmup = warmup;
-    }
-
-    public String getSerialization() {
-        return serialization;
-    }
-
-    public void setSerialization(String serialization) {
-        this.serialization = serialization;
-    }
-
-    @Parameter(key = Constants.TAG_KEY, useKeyAsProperty = false)
-    public String getTag() {
-        return tag;
-    }
-
-    public void setTag(String tag) {
-        this.tag = tag;
-    }
-}
+/*
```

Comment:
Why are you changing all the lines? Please avoid changing the style, and keep only minimal necessary changes.

Reply:
This is wired, seems it changed code style automatically.. I will try to fix it!